test/Parser/testnetcdfParse.m

```diff
@@ -2,7 +2,7 @@
 
     properties (TestParameter)
         anmn_file = files2namestruct(only_netcdf(rdir([toolboxRootPath 'data/testfiles/aodn/ANMN'])))
-        abos_file = files2namestruct(only_netcdf(rdir([toolboxRootPath 'data/testfiles/aodn/ABOS'])))
+        dwm_file = files2namestruct(only_netcdf(rdir([toolboxRootPath 'data/testfiles/aodn/DWM'])))
```

Comment:
you will need to change the directory structure accordingly where the test files are stored.

Reply:
This is already done abd explained in details here:
https://github.com/aodn/PO-Backlog/issues/2185#issuecomment-930797090